examples/gno.land/p/demo/grc/grc20/token.gno

```diff
@@ -1,13 +1,14 @@
 package grc20
 
 import (
-	"math/overflow"
 	"std"
 	"strconv"
 
 	"gno.land/p/demo/ufmt"
 )
 
+const MaxUint64 = uint64(1<<63 - 1)
```

Comment:
question: why you don't use `math.MaxUint64` ?

Reply:
this was to reduce gas consuption through package calls

Reply:
thank you for your answer ^^